docs/manifest.json

```
@@ -158,6 +158,13 @@
 						}
 					]
 				},
+				{
+					"title": "Coder Desktop",
+					"description": "Use Coder Desktop to access your workspace like it's a local machine",
+					"path": "./user-guides/desktop/index.md",
+					"icon_path": "./images/icons/computer-code.svg",
+					"state": ["beta"]
```

```suggestion
					"state": ["alpha"]
```

probably? let's try it